README.md

```diff
@@ -1,70 +1,46 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Europe-Quick-Check App
 
-## Available Scripts
+> Europe-Quick-Check App  allows you to:
 
-In the project directory, you can run:
+- Display a list of European countries
+- Search any European country of your choice
+- Details of each Country.
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Built With
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- React and Redux
 
-### `npm test`
+## Live Demo 
+Feel free to check the [Live link](https://europe-quick-check.herokuapp.com/).
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Video
+[video link](https://drive.google.com/file/d/1iAn6DcECNdvFErQ5Kc5n7IdQFC_kB393/view?usp=sharing).
 
-### `npm run build`
+## Author
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+👤 **Oyewunmi Gbenga**
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- GitHub: [@githubhandle](https://github.com/kelomo2502)
+- Twitter: [@twitterhandle](https://twitter.com/kelomoJs)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/oyewunmi-gbenga)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## 🤝 Contributing
 
-### `npm run eject`
+Contributions, issues, and feature requests are welcome!
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+Feel free to check the [issues page](https://github.com/kelomo2502/europe-quick-check/issues).
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Show your support
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Give a ⭐️ if you like this project!
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## Acknowledgments
 
-## Learn More
+- I acknowledge Original design idea by Nelson Sakwa on Behance.
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you hyperlink the author of the template design `Nelson Sakwa` with his link profile from Behance i.e. `[Nelson Sakwa](https://www.behance.net/sakwadesignstudio)`